addons/mail/models/mail_message.py

```diff
@@ -872,7 +872,7 @@ def _message_reaction(self, content, action):
         group_domain = [("message_id", "=", self.id), ("content", "=", content)]
         count = self.env["mail.message.reaction"].search_count(group_domain)
         group_command = "ADD" if count > 0 else "DELETE"
-        personas = [("ADD" if action == "add" else "DELETE", {"id": guest.id if guest else partner.id, "type": "guest" if guest else "partner"})] if guest or partner else []
+        personas = [("ADD" if action == "add" else "DELETE", {"guestId": guest.id, "partnerId": partner.id})] if guest or partner else []
```

Comment:
should not have guestId and partnerId at the same time ideally (even if one is false), just listing the only key that is used is probably better?